src/Xamarin.Android.Build.Tasks/Utilities/Aapt2Daemon.cs

```diff
@@ -183,15 +213,15 @@ private void Aapt2DaemonStart ()
 					bool errored = false;
 					try {
 						// try to write Unicode UTF8 to aapt2
-						StreamWriter writer = aapt2.StandardInput;
-						foreach (var arg in job.Commands)
-						{
-							writer.WriteLine (arg);
+						using (StreamWriter writer = new StreamWriter (aapt2.StandardInput.BaseStream, MonoAndroidHelper.UTF8withoutBOM, bufferSize: 1024, leaveOpen: true)) {
```

Comment:
Does this change fix the problem on its own? Or does the encoding have to also be set when the process is started?

Reply:
ok so on my machine it fixes it on its own, on CI it does not. So its not 100% effective, so I've left it in for the case where the `Console.InputEncoding` call fails and we need a back up.